### ISSUE TITLE
how to reset passman?

### ISSUE THREAD

**Issue by ib54003** [brantje/passman#132]
Hey,
how can i reset the database?

**Comment by animalillo** [comment]
you mean, empty all users vault, passwords files and config?

**Comment by ib54003** [comment]
yes thats what i mean .

**Comment by ib54003** [comment]
ping

**Comment by animalillo** [comment]
Sorry for the delay, but you can basically trucate all the passman tables and you'll be ready to go

**Comment by morriswinkler** [comment]
not true, i just pulled the latest version from this repro deleted my old tables and i am stuck in a loading cycle.  


commit 3df4e5a0aa91c06b840dc433e5c7ce192e48e199
Author: Marcos Zuriaga <wolfi@wolfi.es>
Date:   Sun Jan 3 18:56:28 2016 +0100

    Missing files from last commit


Owncloud 8.1.1

**Comment by animalillo** [comment]
Have you truncated all the passman tables?
Also, what commit are you on, or passman version?

**Comment by animalillo** [comment]
Deactivate and reactivate the app

El 30 de marzo de 2016 2:46:37 CEST, morriswinkler <notifications@github.com> escribió:
>not true, i just pulled the latest version from this repro deleted my
>old tables and i am stuck in a loading cycle.  
>
>
>commit 3df4e5a0aa91c06b840dc433e5c7ce192e48e199
>Author: Marcos Zuriaga <wolfi@wolfi.es>
>Date:   Sun Jan 3 18:56:28 2016 +0100
>
>    Missing files from last commit
>
>
>Owncloud 8.1.1
>
>---
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/brantje/passman/issues/132#issuecomment-203172923

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.
